client/scss/nav.scss

```diff
@@ -0,0 +1,36 @@
+@import 'https://fonts.googleapis.com/css?family=Press+Start+2P';
```

Comment:
e.g. I might say "do we want to import fonts from Google ... how about we do X 
OR  awesome :fire: